buổi 10/main.js

```diff
@@ -0,0 +1,98 @@
+// Bài 1:
+// Cách 1:
+const getArray = (string, lengthArray) => {
+  const arr = []
+  for(let i = 0; i < lengthArray; i++ ) {
+      arr.push(string)
+  }
+  return arr
+}
+
+//Cách 2:
+const getArray = (string, lengthArray) => {
+	let i = 0 
+	const arr = []
+	do {
+		arr.push(string)
+		i++
+	} while(i < lengthArray)
+	return arr
+}
+
+//Bài 2:
+const reArr = array => {
+  const newArr = []
+  for (let i = 0; i < array.length; i++) {
+    newArr.unshift(arr[i])
+  }
+  return newArr
+}
+
+//Bài 3:
+const deleteArray = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(typeof(array[i]) == 'number' && array[i] !=0) {
```

Comment:
```suggestion
    if (typeof(array[i]) === 'number' && array[i] !== 0) {
```
Các dấu cách vẫn viết rất linh ta linh tinh
Nhìn kỹ cách đặt dấu cách
và dùng `!==` thay cho `!=`, dùng `===` thay cho `==`